pkg/dataset/config.go

```diff
@@ -44,12 +45,12 @@ type Config struct {
 
 // Metrics contains dataset configuration options for metrics data.
 type Metrics struct {
-	ChunkInterval   day.Duration `yaml:"default_chunk_interval"`
-	Compression     *bool        `yaml:"compress_data"` // Using pointer to check if the value was set.
-	HALeaseRefresh  day.Duration `yaml:"ha_lease_refresh"`
-	HALeaseTimeout  day.Duration `yaml:"ha_lease_timeout"`
-	RetentionPeriod day.Duration `yaml:"default_retention_period"`
-	Downsample      *Downsample  `yaml:"downsample,omitempty"`
+	ChunkInterval   day.Duration   `yaml:"default_chunk_interval"`
+	Compression     *bool          `yaml:"compress_data"` // Using pointer to check if the value was set.
+	HALeaseRefresh  day.Duration   `yaml:"ha_lease_refresh"`
+	HALeaseTimeout  day.Duration   `yaml:"ha_lease_timeout"`
+	RetentionPeriod day.Duration   `yaml:"default_retention_period"`
+	Downsample      *rollup.Config `yaml:"downsample,omitempty"`
```

Comment:
This was changed [here](https://github.com/timescale/promscale/pull/1737/files#diff-b16f8ee862c8f1f5b69f5f1856d60cb38c393083834de0446cdd238b94e94f53R45-R49) so maybe it would be good if you rebase your feature branch, since this might give you conflicts later.

Reply:
I am not confident of rebasing the branch ATM, given there might be more conflicts than this. I think it will be better to resolve conflicts later on when I dedicate 1 day for it, than doing in between. This way the benchmarks also behave stable for rollups development only. The branch is already behind 29 commits now :)